packages/fxa-auth-db-mysql/lib/db/schema/patch-099-100.sql

```diff
@@ -0,0 +1,22 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('99');
+
+CREATE PROCEDURE `readSecurityEvents_1` (
+    IN inUid BINARY(16)
+)
+BEGIN
+    SELECT
+        n.name,
+        e.verified,
+        e.createdAt
+    FROM
+        securityEvents e
+    LEFT JOIN securityEventNames n
+        ON e.nameId = n.id
+    WHERE
+        e.uid = inUid
+    ORDER BY e.createdAt DESC;
```

Comment:
I suspect this list can end up being pretty big, I wonder whether we should limit by 50 like the other version until we better understand the behavior of the overall feature.

Reply:
But it will fetch for only 1 user's uid, so how can the list get big ?

Reply:
I think so, let's start there.